Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Use Node.js runtime as builder base image.
 # Since the built app is platform-agnostic, we use the host architecture
 # even if we are cross-building the image.
-FROM --platform=$BUILDPLATFORM node:21 AS builder
```

Comment:
Can we keep this? If your Docker version doesn't support this, you may be running on a version without BuildKit support (or you may have to use `docker buildx`). Having this speeds up our cross-compilation Docker CI builds significantly since the aarch64 build won't have to run npm in QEMU

Reply:
Ah sorry, I removed it for testing on my machine